styles/helpers/bg-bleed.css

```diff
@@ -0,0 +1,9 @@
+.bg-bleed {
```

Comment:
I've suggested this before but the bleed term seems confusing, can we just change this to `bg-max` or `container-max`, something with max seems more preferred since thats exactly what its doing.